.eslintrc

```diff
@@ -65,6 +65,7 @@
             "named": "never",
             "asyncArrow": "always"
         }],
+        "array-bracket-spacing": "off",
```

Comment:
Just like previous PR https://github.com/eBay/ebayui-core-react/pull/146/files#r1018759202, better to have this on separated PR.